pyins/integrate.py

```diff
@@ -343,3 +351,151 @@ def integrate_stationary(dt, lat, Cnb, theta, dv):
     integrate_fast_stationary(dt, lat, Cnb_arr, V_arr, theta, dv)
 
     return Cnb_arr, V_arr
+
+
+@jit
+def __dcm_from_rv_single(rv, out=None):
+    rv1, rv2, rv3 = rv
+    rv11 = rv1 * rv1
+    rv12 = rv1 * rv2
+    rv13 = rv1 * rv3
+    rv22 = rv2 * rv2
+    rv23 = rv2 * rv3
+    rv33 = rv3 * rv3
+
+    norm2 = rv11 + rv22 + rv33
+    if norm2 > 1e-6:
+        norm = norm2 ** 0.5
+        k1 = math.sin(norm) / norm
+        k2 = (1.0 - math.cos(norm)) / norm2
+    else:
+        norm4 = norm2 * norm2
+        k1 = 1.0 - norm2 / 6.0 + norm4 / 120.0
+        k2 = 0.5 - norm2 / 24.0 + norm4 / 720.0
+
+    if out is None:
+        out = np.empty((3,3))
+    out[0, :] = 1.0 - k2*(rv33 + rv22), -k1*rv3 + k2*rv12, k1*rv2 + k2*rv13
+    out[1, :] = k1*rv3 + k2*rv12, 1.0 - k2*(rv33 + rv11), -k1*rv1 + k2*rv23
+    out[2, :] = -k1*rv2 + k2*rv13, k1*rv1 + k2*rv23, 1.0 - k2*(rv22 + rv11)
+    return out
+
+
+@jit
+def __mv_dot3(A, b):
```

Comment:
Names are too cryptic for no reason.

Use of __ and _ doesn't make sense in my opinion, use simply _ to indicated module-private function.